Makefile

```diff
@@ -24,24 +24,24 @@ build_race:
 dist:
 	@BUILD_TAGS='$(BUILD_TAGS)' sh -c "'$(CURDIR)/scripts/dist.sh'"
 
-test:
-	@echo "--> Running linter"
-	@make metalinter_test
-	@echo "--> Running go test"
-	@go test $(PACKAGES)
+test: tools get_vendor_deps
+	echo "--> Running linter"
+	make metalinter_test
+	echo "--> Running go test"
+	go test $(PACKAGES)
 
 test_race:
-	@echo "--> Running go test --race"
-	@go test -v -race $(PACKAGES)
+	echo "--> Running go test --race"
```

Comment:
why remove `@`. the output will be

echo "--> Running go test --race"
--> Running go test --race

I don't think we want that. If you don't like `@`, then we can remove `echo` statements altogether.

Reply:
i got carried away debugging, will revert